src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -1407,36 +1407,35 @@ private async Task<bool> OnEvaluateOnCallFrame(MessageId msg_id, int scopeId, st
             return true;
         }
 
-        internal async Task<Result> GetScopeProperties(SessionId msg_id, int scopeId, CancellationToken token)
+        internal async Task<GetMembersResult> GetScopeProperties(SessionId msg_id, int scopeId, CancellationToken token)
         {
             try
             {
                 ExecutionContext context = GetContext(msg_id);
                 Frame scope = context.CallStack.FirstOrDefault(s => s.Id == scopeId);
                 if (scope == null)
-                    return Result.Err(JObject.FromObject(new { message = $"Could not find scope with id #{scopeId}" }));
+                    throw new Exception($"Could not find scope with id #{scopeId}");
```

Comment:
future work: It's fine for this PR, but we need to improve the exception types we use, so we can handle them a bit better.